com.unity.render-pipelines.core/Editor/Lighting/ProbeVolume/ProbeGIBaking.cs

```diff
@@ -308,6 +308,8 @@ internal static void PerformDilation()
 
             foreach (var sceneData in perSceneDataList)
             {
+                if (!ProbeReferenceVolume.instance.sceneData.SceneHasProbeVolumes(sceneData.gameObject.scene)) continue;
```

Comment:
isn't it possible to destroy all perSceneData objects of scenes that are invalid at start of baking ? I suspect we will forget to call this `SceneHasProbeVolumes` at some point and end up with the same issue

Reply:
Yes, doable after this https://github.com/Unity-Technologies/Graphics/pull/6907 PR lands.